_includes/head.html

```diff
@@ -2,7 +2,7 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>{% if page.title %}{{ page.title | escape }}{% else %}{{ site.title | escape }}{% endif %}</title>
+  <title>{% if page.title %}{{ page.title | escape }} | {{site.title}} {% else %}{{ site.title | escape }}{% endif %}</title>
```

Comment:
Why not replace this with ` page.title | | site.title` basically? (need to look up correct syntaxt)

Reply:
The idea is to improve SEO by having both the page and site title together. the pipe character appears in the title